website/events/templates/events/event.html

```diff
@@ -239,17 +239,27 @@ <h1 class="text-center section-title">{{ event.title }}</h1>
                             </td>
                         </tr>
 
-                        {% if event.food_event %}
-                            <tr>
-                                <th>
-                                    Pizza
-                                </th>
-                                <td>
-                                    <a href="{% url "pizzas:index" %}" class="btn btn-primary">
-                                        {% trans "Order" context "pizzas" %}
-                                    </a>
-                                </td>
-                            </tr>
+                        {% if event.food_event %} 
+                            {% if event.food_event == event.food_event.current or event.food_event.just_ended or event.food_event.in_the_future %}
+                                <tr>
+                                    <th>
+                                        Pizza
+                                    </th>
+                                    <td>
+                                    {% if event.food_event == event.food_event.current or event.food_event.just_ended %}
+                                        <a href="{% url "pizzas:index" %}" class="btn btn-primary">
+                                            {% trans "Order" context "pizzas" %}
+                                        </a>
+                                    {% elif event.food_event.in_the_future %}
+                                        <div class="d-inline-block" data-toggle="tooltip" data-placement="auto" title="Ordering pizza for this event will be possible starting {{ event.food_event.start }}">
```

Comment:
Please keep the trans tag for consistency.

Reply:
Like how I did it now?

Reply:
perfect